azkaban-common/src/test/java/azkaban/utils/EmailerTest.java

```diff
@@ -115,6 +119,26 @@ public void testSendErrorEmail() throws Exception {
         .isEqualToIgnoringWhitespace(this.message.getBody());
   }
 
+  @Test
+  public void alertOnFailedUpdate() throws Exception {
```

Comment:
Consider the name `testAlertOnFailedUpdate()`?

Reply:
I'd rather not change this. I believe prefixing with `test` doesn't add value. I've seen for example @HappyRay also omit such prefix in some new tests he's created.

Reply:
👍